src/Sinch/Numbers/NumberPattern.cs

```diff
@@ -18,37 +18,40 @@ public class NumberPattern
         /// <summary>
         ///     Search pattern to apply. The options are, START, CONTAIN, and END.
         /// </summary>
-        public SearchPattern? SearchPattern { get; set; }
+        public SearchPattern SearchPattern { get; set; }
 
         internal IEnumerable<KeyValuePair<string, string>> GetQueryParamPairs()
         {
             var list = new List<KeyValuePair<string, string>>();
             list.Add(new KeyValuePair<string, string>("numberPattern.pattern", Pattern));
-            if (SearchPattern.HasValue)
+            if (SearchPattern is not null)
```

Comment:
Not seeing from OAS file this field is optional if `numberPattern.pattern` is set.
Side note, not related to this PR content but for this file:
https://github.com/sinch/sinch-sdk-dotnet/blob/b1cdc555e361b34a8f71639969302fff4fb49819/src/Sinch/Numbers/NumberPattern.cs#L50
There is an inconsistency with doc and OAS vs API implementation.
Current API implementation is using the `CONTAINS` keyword (not `CONTAIN`
When using `CONTAIN`, server will return:
```javascript
"error": {
  "code": 400,
  "message": "parsing field \"search_pattern\": \"CONTAIN\" is not a valid value", 
  "status": "INVALID_ARGUMENT",
  "details": [] 
 }
}
``` 
Ticket opened for API: https://tickets.sinch.com/browse/NUM-5022

Reply:
They are both optional and independent